src/_N001_Email.cpp

```diff
@@ -193,34 +193,41 @@ bool NPlugin_001_send(const NotificationSettingsStruct& notificationsettings, co
 
 
     // Use Notify Command's destination email address(s) if provided in Command rules.
-    // Sample Rule: Notify 1, "{email1@domain.com;email2@domain.net},Test email from %sysname%.<br/> How are you?<br/>Have a good day.<br/>"
+    // Sample Rule: Notify 1, "{email1@domain.com;email2@domain.net}Test email from %sysname%.<br/> How are you?<br/>Have a good day.<br/>"
     String subAddr = "";
+    String tmp_ato = "";
     int pos_brace1 = aMesg.indexOf('{');
     int pos_amper  = aMesg.indexOf('@');
     int pos_brace2 = aMesg.indexOf('}');
     if(pos_brace1 == 0 && pos_amper > pos_brace1 && pos_brace2 > pos_amper) {
         subAddr = aMesg.substring(pos_brace1+1, pos_brace2);
+        subAddr.trim();
+        tmp_ato = subAddr;
         # ifndef BUILD_NO_DEBUG
-        addLog(LOG_LEVEL_DEBUG, strformat(F("Email: Notify substitute email addr: %s"), subAddr.c_str())); // TEB
+        addLog(LOG_LEVEL_DEBUG, strformat(F("Email: Substitute Receiver (ato): %s"), subAddr.c_str())); // TEB
         # endif

```

Comment:
In general, for the builds that _do_ have DEBUG logging enabled, this will process the `strformat()` function, but discard the result in `addLog()` when logging level is not set to DEBUG, i.e. nearly always.
Best practice is to wrap an `if (logLevelActiveFor(LOG_LEVEL_DEBUG)) { ... }` condition around it, reducing CPU-cycles a bit.